homeassistant/components/sensor/waze_travel_time.py

```diff
@@ -46,34 +49,49 @@
     vol.Optional(CONF_EXCL_FILTER): cv.string,
 })
 
+TRACKABLE_DOMAINS = ['device_tracker', 'sensor', 'zone']
+
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Waze travel time sensor platform."""
-    destination = config.get(CONF_DESTINATION)
-    name = config.get(CONF_NAME)
-    origin = config.get(CONF_ORIGIN)
-    region = config.get(CONF_REGION)
-    incl_filter = config.get(CONF_INCL_FILTER)
-    excl_filter = config.get(CONF_EXCL_FILTER)
+    def run_setup(event):
+        """Set up the waze travel time sensor."""
+        destination = config.get(CONF_DESTINATION)
+        name = config.get(CONF_NAME)
+        origin = config.get(CONF_ORIGIN)
+        region = config.get(CONF_REGION)
+        incl_filter = config.get(CONF_INCL_FILTER)
+        excl_filter = config.get(CONF_EXCL_FILTER)
 
-    try:
-        waze_data = WazeRouteData(
-            origin, destination, region, incl_filter, excl_filter)
-    except requests.exceptions.HTTPError as error:
-        _LOGGER.error("%s", error)
-        return
+        sensor = WazeTravelTime(hass, name, origin, destination, region,
+                                incl_filter, excl_filter)
+        add_devices([sensor])
 
-    add_devices([WazeTravelTime(waze_data, name)], True)
+    hass.bus.listen_once(EVENT_HOMEASSISTANT_START, run_setup)
```

Comment:
Why wait with adding the entity until home assistant start? Couldn't we add the entity during setup and just make sure `update` is called when home assistant is started?